skbeam/core/accumulators/binned_statistic.py

```diff
@@ -196,10 +208,15 @@ def __call__(self, values):
 
         Returns
         -------
-        statistic : array
+        statistic_values : array
             The values of the selected statistic in each bin.
         """
 
+        self.statistic = statistic
```

Comment:
Move this down one line. Needs to be after the guard against None but before you check to see if it is in that list